Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,278 @@
+//
+//  ContentView.swift
+//  testCustomEntitlementsComputation
+//
+//  Created by Andrés Boedo on 4/20/23.
+//
+
+import SwiftUI
+import RevenueCat
+
+struct CustomerInfoStreamFires: Identifiable {
+    let customerInfo: CustomerInfo
+    let id = UUID()
+    let date: Date
+}
+
+struct ContentView: View {
+
+    @State private var streamTask: Task<Void, Never>?
+    @State private var offerings: Offerings?
+    @State private var customerInfoStreamFires: [CustomerInfoStreamFires] = []
+    @State private var showingLogInAlert = false
+    @State private var appUserID = Constants.defaultAppUserID
```

Comment:
I guess it could be useful to be able to select the initial app user id before configuring the SDK... But not needed for V1 of this 👍

Reply:
Agreed. I didn't do it to keep this as simple as possible, since having a way to set up an appUserID would almost instantly mean having some navigation where you start from the configuration screen then go to a second screen and we're trying to keep the scope as small as possible here